Dockerfile

```diff
@@ -7,13 +7,13 @@ ADD collectd.conf /etc/collectd/
 ADD entrypoint.sh /entrypoint.sh
 
 RUN echo 'ef97838e249814ad96a2edc0410326efbb4f3d08  collectd-5.5.0.tar.bz2' > /tmp/collectd.sig && \
-    apt-get update && \
-    apt-get install -y build-essential wget tar lbzip2 && \
+    apt-get update && apt-get install -y build-essential wget tar lbzip2 && \
```

Comment:
@adamdecaf so you are installing build-essentials and then removing it after you configure collectd?